xclim/indices/run_length.py

```diff
@@ -293,6 +294,160 @@ def last_run(
     return out
 
 
+def run_length_with_date(
+    da: xr.DataArray, window: int, date: str = "07-01", dim: str = "time",
+):
+    """Return the index of the last item of the last run of at least a given length.
+
+    Parameters
+    ----------
+    da : xr.DataArray
+      Input N-dimensional DataArray (boolean)
+    window : int
+      Minimum duration of consecutive run to accumulate values.
+    date:
+      The date that a run must include to be considered valid.
+    dim : str
+      Dimension along which to calculate consecutive run (default: 'time').
+
+    Returns
+    -------
+    out : xr.DataArray
+      Index (or coordinate if `coord` is not False) of last item in last valid run. Returns np.nan if there are no valid run.
```

Comment:
I believe there is some forgotten copypasta here ;).

Reply:
Fixed.